atomic_reactor/inner.py

```diff
@@ -343,31 +355,26 @@ def build_docker_image(self):
                 self.autorebuild_canceled = True
                 raise
 
-            start_time = datetime.datetime.now()
-            self.plugins_timestamps['dockerbuild'] = start_time.isoformat()
-
-            build_result = self.builder.build()
-
+            logger.info("running buildstep plugins")
+            buildstep_runner = BuildStepPluginsRunner(self.builder.tasker, self,
+                                                     self.buildstep_plugins_conf,
+                                                     plugin_files=self.plugin_files)
```

Comment:
atomic_reactor/inner.py:360:54: E128 continuation line under-indented for visual indent
atomic_reactor/inner.py:361:54: E128 continuation line under-indented for visual indent